app/controllers/CampaignsController.scala

```diff
@@ -16,13 +18,35 @@ class CampaignsController(
   components: ControllerComponents,
   stage: String,
   runtime: zio.Runtime[ZEnv],
-  dynamo: DynamoChannelTests
+  dynamoChannelTests: DynamoChannelTests,
+  dynamoCampaigns: DynamoCampaigns
 )(implicit ec: ExecutionContext)
   extends S3ObjectController[Campaigns](authAction, components, stage, filename = "campaigns.json", runtime) {
```

Comment:
I've just realised we can now remove this line. This controller no longer needs to interact with S3 (by inheriting from the `S3ObjectController`)

Reply:
It seems that it works if instead of `S3ObjectController` we extend `AbstractController with Circe with LazyLogging` and implement a private `run` function (as in `ChannelTestsController`). Simply deleting the above line will throw errors.

Reply:
ah yes, that sounds right!

Reply:
Also, if I go ahead with these changes, the `controllers.CampaignsController.set` is broken, so I assume we'd need to add a `set` function as well to the controller (would this be similar to `setStatus` in `ChannelTestsController`?).

Reply:
The endpoint using `controllers.CampaignsController.set` can also be removed. It was for POSTing the full array of campaigns to S3. That endpoint is no longer used now that you've removed the call to `saveSettings` in the client code